components/list/list-item-mixin.js

```diff
@@ -319,8 +326,9 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			:host([_hovering-selection]) [slot="outside-control-container"],
 			:host([_focusing-primary-action]) [slot="outside-control-container"],
 			:host(:not([selection-disabled]):not([skeleton])[selected][_hovering-selection]) [slot="outside-control-container"],
-			:host(:not([selection-disabled]):not([skeleton])[selectable][_focusing]) [slot="outside-control-container"] {
-				border-color: #b6cbe8; /* celestine alpha 0.3 */
+			:host(:not([_list-item-new-styles]):not([selection-disabled]):not([skeleton])[selectable][_focusing]) [slot="outside-control-container"],
+			:host([_list-item-new-styles]:not([selection-disabled]):not([button-disabled]):not([skeleton])[_focusing]) [slot="outside-control-container"] {
```

Comment:
The feature flag off rule is the same as before. The feature flag on rule does the following:
- Removes requirement that item is `selectable` to show the full border
- Adds condition that item does not have `button-disabled`